kordac/KordacExtension.py

```diff
@@ -72,6 +59,7 @@ def extendMarkdown(self, md, md_globals):
                 md.parser.blockprocessors.add(tag_processor[0], tag_processor[1], tag_processor[2])
 
     def reset(self):
```

Comment:
Good thinking, I'll think of a better name.

On that note, should the function `default_tags` be called `tag_defaults`, as the first one could be interpreted as verb-noun, as in default the tags?